x-pack/test_serverless/functional/test_suites/observability/cases/view_case.ts

```diff
@@ -0,0 +1,965 @@
+
+      it('filters by all by default', async () => {
+        const allBadge = await find.byCssSelector(
+          '[data-test-subj="user-actions-filter-activity-button-all"] span.euiNotificationBadge'
+        );
+
+        expect(await allBadge.getAttribute('aria-label')).equal('1 active filters');
+      });
+
+      it('filters by comment successfully', async () => {
+        const commentBadge = await find.byCssSelector(
+          '[data-test-subj="user-actions-filter-activity-button-comments"] span.euiNotificationBadge'
+        );
+
+        expect(await commentBadge.getAttribute('aria-label')).equal('0 available filters');
+
+        const commentArea = await find.byCssSelector(
+          '[data-test-subj="add-comment"] textarea.euiMarkdownEditorTextArea'
+        );
+        await commentArea.focus();
+        await commentArea.type('Test comment from automation');
+        await testSubjects.click('submit-comment');
+
+        await header.waitUntilLoadingHasFinished();
+
+        await testSubjects.click('user-actions-filter-activity-button-comments');
+
+        expect(await commentBadge.getAttribute('aria-label')).equal('1 active filters');
+      });
+
+      it('filters by history successfully', async () => {
+        const historyBadge = await find.byCssSelector(
+          '[data-test-subj="user-actions-filter-activity-button-history"] span.euiNotificationBadge'
+        );
+
+        expect(await historyBadge.getAttribute('aria-label')).equal('1 available filters');
+
+        await cases.common.selectSeverity(CaseSeverity.MEDIUM);
+
+        await cases.common.changeCaseStatusViaDropdownAndVerify(CaseStatuses['in-progress']);
+
+        await header.waitUntilLoadingHasFinished();
+
+        await testSubjects.click('user-actions-filter-activity-button-history');
+
+        expect(await historyBadge.getAttribute('aria-label')).equal('3 active filters');
+      });
+
+      it('sorts by newest first successfully', async () => {
+        await testSubjects.click('user-actions-filter-activity-button-all');
+
+        const AllBadge = await find.byCssSelector(
+          '[data-test-subj="user-actions-filter-activity-button-all"] span.euiNotificationBadge'
+        );
+
+        expect(await AllBadge.getVisibleText()).equal('4');
+
+        const sortDesc = await find.byCssSelector(
+          '[data-test-subj="user-actions-sort-select"] [value="desc"]'
+        );
+
+        await sortDesc.click();
+
+        await header.waitUntilLoadingHasFinished();
+
+        const userActionsLists = await find.allByCssSelector(
+          '[data-test-subj="user-actions-list"]'
+        );
+
+        const actionList = await userActionsLists[0].findAllByClassName('euiComment');
+
+        expect(await actionList[0].getAttribute('data-test-subj')).contain('status-update-action');
+      });
+    });
+
+    // TO FIX - https://github.com/elastic/kibana/issues/164736
+    describe.skip('Lens visualization', () => {
```

Comment:
I believe this is fine because these tests are failing only in serverless.

Reply:
This issue is fixed and merged in PR: https://github.com/elastic/kibana/pull/165225. So you can unskip and it should work as expected.